rally/challenges/default.json

```diff
@@ -65,4 +65,65 @@
           "clients": 1
         } 
       ]
+    },
+    {
+      "name": "query-apm",
+      "default": false,
+      "description": "Measure different APM related queries used from the APM UI. Indexes APM events into multiple indices, each representing one day, as a preparation step for queries.",
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": "create-index"
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": "check-cluster-health"
+        },
+        {% set comma = joiner() %}
+        {% for item in range(index_count) %}
+        {{ comma() }}
+        {
+          "parallel": {
+            "tasks": [
+              {
+                "operation": "index-append-error-{{item}}",
+                "clients": {{bulk_indexing_clients | default(1)}}
+              },
+              {
+                "operation": "index-append-transaction-{{item}}",
+                "clients": {{bulk_indexing_clients | default(3)}}
+              },
+              {
+                "operation": "index-append-span-{{item}}",
+                "clients": {{bulk_indexing_clients | default(6)}}
+              }    
+            ]
+          }
+        }
+        {% endfor %},
+        {
+          "operation": "force-merge",
+          "clients": 1
+        },
+        {
+          "operation": "query-all-spans",
```

Comment:
Throughput is not throttled for any of the queries which is unusual. While this setup can provide some value, I think it is unrealistic to assume that the system would receive queries at the highest possible rate (see our [docs](https://esrally.readthedocs.io/en/stable/faq.html#what-does-latency-and-service-time-mean-and-how-do-they-related-to-the-took-field-that-elasticsearch-returns) for more background info).

Reply:
Thanks for the note, I added a throughput parameter now and set it to a reasonable number.